frontend/www/js/omegaup/components/user/Edit.vue

```diff
@@ -0,0 +1,408 @@
+                  <input
+                    v-model="oldPassword"
+                    name="name"
+                    value=""
+                    required
+                    type="password"
+                    size="30"
+                    class="form-control"
+                  />
+                </div>
+              </div>
+              <div class="form-group">
+                <label for="name" class="col-md-4 control-label">{{
+                  T.userEditChangePasswordNewPassword
+                }}</label>
+                <div class="col-md-7">
+                  <input
+                    v-model="newPassword1"
+                    name="name"
+                    value=""
+                    required
+                    type="password"
+                    size="30"
+                    class="form-control"
+                  />
+                </div>
+              </div>
+              <div class="form-group">
+                <label for="name" class="col-md-4 control-label">{{
+                  T.userEditChangePasswordRepeatNewPassword
+                }}</label>
+                <div class="col-md-7">
+                  <input
+                    v-model="newPassword2"
+                    name="name"
+                    value=""
+                    required
+                    type="password"
+                    size="30"
+                    class="form-control"
+                  />
+                </div>
+              </div>
+
+              <div class="col-md-offset-6 col-md-6 col-xs-12">
+                <button
+                  type="submit"
+                  class="btn btn-primary col-xs-offset-1 col-xs-5"
+                >
+                  {{ T.wordsSaveChanges }}
+                </button>
+                <a href="/profile/" class="btn col-xs-5 btn-cancel">{{
+                  T.wordsCancel
+                }}</a>
+              </div>
+            </form>
+          </div>
+        </form>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import T from '../../lang';
+import user_MainInfo from './MainInfo.vue';
+import DatePicker from '../DatePicker.vue';
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
+import user_ManageIdentities from './ManageIdentities.vue';
+import * as iso3166 from '@/third_party/js/iso-3166-2.js/iso3166.min.js';
+import * as typeahead from '../../typeahead';
+import common_Typeahead from '../common/Typeahead.vue';
+import { types } from '../../api_types';
+@Component({
+  components: {
+    'font-awesome-icon': FontAwesomeIcon,
+    'omegaup-common-typeahead': common_Typeahead,
+    'omegaup-datepicker': DatePicker,
+    'omegaup-user-manage-identities': user_ManageIdentities,
+    'omegaup-user-maininfo': user_MainInfo,
+  },
+})
+export default class UserProfileEdit extends Vue {
+  @Prop() data!: types.UserProfileEditDetailsPayload | null;
```

Comment:
Aquí el payload siempre se debería recibir con datos, quiere decir que nunca recibiríamos `null`.
```suggestion
  @Prop() data!: types.UserProfileEditDetailsPayload;
```